Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG FROM_IMAGE=lcskrishna/rocm-pytorch:rocm3.3_ubuntu16.04_py3.6_pytorch_updated
```

Comment:
Do we need an updated built pytorch docker? Anything we need not in already built rocm/pytorch dockers?

Reply:
Yes, the GeneratedFileCleaner api in hipify script is not present in our pre-built docker.